scylla/src/statement/prepared_statement.rs

```diff
@@ -202,6 +202,11 @@ impl PreparedStatement {
     pub fn get_timestamp(&self) -> Option<i64> {
         self.config.timestamp
     }
+
+    /// Adopts configuration from another source
+    pub fn inherit_config(&mut self, config: StatementConfig) {
```

Comment:
I think this method should be `pub(crate)`. I don't think that StatementConfig is a part of the public interface (maybe we export it accidentally, but I'm not sure we should be doing that).

Reply:
Indeed, it's expected for internal use only